src/components/Input/Select.md

```diff
@@ -0,0 +1,67 @@
+### Default Style
+
+```jsx
+<Select type="text" name="text-demo" label="Select Input" note="This is a note." margin={{ bottom: 'md' }}>
+    <option>Choose an option...</option>
+    <option value="Option One">Option One</option>
+    <option value="Option One">Option Two</option>
```

Comment:
The values here should probably be updated to match their text.

Reply:
I copied  the options directly from the Rivet site. I started to do them significantly differently (i.e., with meaningful examples), but wasn't sure how much we wanted to match their examples, so I thought I'd start there.

Reply:
Sounds like it's time to file a bug against Rivet. 😜